src/pages/LoanFinding/LoanFinding.vue

```diff
@@ -159,6 +161,7 @@ export default {
 					client.query({ query: userInfoQuery }),
 					client.query({ query: experimentAssignmentQuery, variables: { id: FIVE_DOLLARS_NOTES_EXP } }),
 					client.query({ query: experimentAssignmentQuery, variables: { id: FLSS_ONGOING_EXP_KEY } }),
+					client.query({ query: experimentAssignmentQuery, variables: { id: QUICK_FILTERS_MOBILE_EXP_KEY } }),
```

Comment:
We don't technically need this here (we run all assignments globally on page load), but it can stay for now.